torch/_functorch/partitioners.py

```diff
@@ -106,21 +107,25 @@ class MinCutOptions:
 
 
 def must_recompute(node: fx.Node) -> bool:
```

Comment:
Actually, based off of some conversation we had earlier today, I think from the compiler POV, we always want to treat both `MUST_RECOMPUTE` and `PREFER_RECOMPUTE` as hard tags.

Reply:
So change this to `node.meta.get... == MUST_RECOMPUTE or PREFER_RECOMPUTE`.